packages/core/src/domain/session/sessionStore.ts

```diff
@@ -40,13 +47,20 @@ export function startSessionStore<TrackingType extends string>(
   const renewObservable = new Observable<void>()
   const expireObservable = new Observable<void>()
 
+  ;(window as BrowserWindow).DD_SDK_AUDIT = (window as BrowserWindow).DD_SDK_AUDIT || []
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I don't think it's a good idea to introduce a new global variable on customer websites. What about using a feature flag?

Reply:
👍
I'll put it behind `cookie-lock`